<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EchoSignal</class>
 <widget class="QDialog" name="EchoSignal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Параметры эхосигнала</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Количество точек расчета</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="numDotBox">
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="labelParam_7">
         <property name="text">
          <string>Угол места цели, град</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="elevationBox">
         <property name="maximum">
          <double>60000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="labelParam_4">
         <property name="text">
          <string>Азимут цели, град</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="azimuthBox">
         <property name="maximum">
          <double>60000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="labelParam_5">
         <property name="text">
          <string>Относительная скорость, м/с</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="relativeSpedBox">
         <property name="maximum">
          <double>60000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="labelParam_6">
         <property name="text">
          <string>Эквивалентный радиус цели, м</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="equivalentRadBox">
         <property name="maximum">
          <double>60000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lablelChannel_1">
       <property name="text">
        <string>Номер канала</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="channel1Box">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="pushButtonCompute">
       <property name="text">
        <string>Расcчитать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelParam_1">
         <property name="text">
          <string>labelParam_1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="param1Box">
         <property name="maximum">
          <double>60000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelParam_2">
         <property name="text">
          <string>labelParam_2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="param2Box">
         <property name="maximum">
          <double>60000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labelParam_3">
         <property name="text">
          <string>labelParam_3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="param3Box">
         <property name="maximum">
          <double>60000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
